metadata-service/war/src/main/java/com/linkedin/gms/servlet/OpenAPIServletConfig.java

```diff
@@ -18,7 +18,8 @@
       "io.datahubproject.openapi.v1",
       "io.datahubproject.openapi.v2",
       "io.datahubproject.openapi.v3",
-      "com.linkedin.gms.factory.timeline"
+      "com.linkedin.gms.factory.timeline",
+      "org.springdoc"
```

Comment:
Curious, why was this necessary?

Reply:
The `org.springdoc` package includes the `@Controller` classes that create the swagger UI endpoints. Without Spring able to detect those controllers they never get wired up.